experimental/devices/sn3218/sn3218_test.go

```diff
@@ -0,0 +1,235 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package sn3218
+
+import (
+	"bytes"
+	"testing"
+
+	"periph.io/x/periph/conn/i2c/i2ctest"
+)
+
+func setup() *i2ctest.Record {
+	bus := i2ctest.Record{
+		Bus: nil,
+		Ops: []i2ctest.IO{},
+	}
+	return &bus
+}
+
+func TestNew(t *testing.T) {
+	bus := i2ctest.Record{
+		Bus: nil,
+		Ops: []i2ctest.IO{},
+	}
+	_, err := New(&bus)
+	if err != nil {
+		t.Fatal("New should not return error", err)
+	}
+	if len(bus.Ops) > 1 {
```

Comment:
In practice, you should use i2ctest.Playback in all of these, not Record.

Reply:
As the device is writeOnly and does not return anything, I decided to use Record. Happy to change it if really needed, but I would like to save the time here.